### PR TITLE
#700 added one line

### PR DESCRIPTION
**Pull request by Kumaranupam2020**
I added one line in supplemental resources section.the freecodecamp is also preffered to learn from scratch

### CODE REVIEW

File: html/html.md [hunk 1]
```diff
@@ -72,3 +72,4 @@ Instructor will do a code review with each apprentice once they've completed the
 - [Intermediate HTML/CSS (Girl Develop It)](https://www.girldevelopit.com/materials/inter-html)
 - [Hostinger:how to add favicons to your site](https://www.hostinger.in/tutorials/how-to-add-favicon-to-website)
 - [NPM favicon package](https://www.npmjs.com/package/favicons)
+- [HTML from scratrch](https://www.freecodecamp.org)
```
Comment (manaswinidas) [inline html/html.md]: Just a typo in 'scratch' otherwise LGTM

**Comment by Kundan28** [comment]
@Kumaranupam2020 I have already told you earlier about tagging the issue in the PR description but you have not done that again...so please do that so we can easily review the PR.

File: html/html.md (hunk 1 above)
Comment (alodahl) [reply html/html.md]: Is there a more direct url than just 'freecodecamp.org'?

**Comment by alodahl** [comment]
@Kumaranupam2020 are you able to fix this PR?  If I don't hear from you in a few days, I will close it.

**Comment by Kumaranupam2020** [comment]
Yeah.....i will see it on coming saturday.

**Comment by Kundan28** [comment]
Any update ? @Kumaranupam2020

File: html/html.md (hunk 1 above)
Comment (Kundan28) [reply html/html.md]: https://learn.freecodecamp.org/responsive-web-design/basic-html-and-html5

**Comment by alodahl** [comment]
Please read the issue description if you would like to work on an issue.  If you fix typos on this PR, we can merge it, but you will not get issue points, @Kumaranupam2020

**Comment by Kundan28** [comment]
I removed this from scoresheet too.

**Pull request closed without merging**